docs/interpreter/jdbc.md

```diff
@@ -240,7 +244,7 @@ You can leverage [Zeppelin Dynamic Form](../usage/dynamic_form/intro.html) insid
 %jdbc_interpreter_name
 SELECT name, country, performer
 FROM demo.performers
-WHERE name='{{"{{performer=Sheryl Crow|Doof|Fanfarlo|Los Paranoia"}}}}'
+WHERE name='${performer=Sheryl Crow|Doof|Fanfarlo|Los Paranoia}'
```

Comment:
I think this is different, right, why this change?

Reply:
Yes, this is a different thing. I have mentioned under **What is this PR for?**:

> I have also taken the liberty to correct a preexisting error in the description of the use of
> Dynamic Forms in the associated documentation (jdbc.md).

Since I was editing _jdbc.md_, I thought correcting the error would be useful for readers. The changed text works as intended under the current implementation, and is also according to the documentation for Dynamic Forms.

Please let me know if corrections should be handled in a different way. 

Thanks for noticing this update.

Reply:
ok thanks